Remove "sessionSavePath property (property)"

marksmithhfx opened the pull request:
The reference contained duplicate information that was not properly formatted. The incorrect information was removed. The string "sessionSavePath (property)" exists in the reference, and is correctly formatted.

marksmithhfx committed:
Remove "sessionSavePath property (property)"

The reference contained duplicate information that was not properly formatted. The incorrect information was removed. The string "sessionSavePath (property)" exists in the reference, and is correctly formatted.

livecodeali commented on docs/dictionary/command/stop-session.lcdoc:
```diff
@@ -20,6 +20,6 @@ Use the <stop session> command to end the session explicitly and save the conten
 
 The location where the $_SESSION array is stored is configured using the <sessionSavePath property>.
```
@marksmithhfx You'll need to change the reference here to `<sessionSavePath> property` so that it links correctly.

livecodeali replied:
To be clear, this refers to `<sessionSavePath property>` on line 21 - this should be `<sessionSavePath> property`, so that the text reads 

```
sessionSavePath property
```

but the link is only on the text `sessionSavePath`

As it is at the moment, `<sessionSavePath property>` will look for an entry called `sessionSavePath property`, not an entry called `sessionSavePath`.

marksmithhfx committed:
Update: correct line 21

On line 21 the text read <sessionSavePath property> not <sessionSavePath> property. As it was LC would look for a reference to "sessionSavePath property" instead of "sessionSavePath". This was corrected by editing the braces.

marksmithhfx commented:
Ali, have updated as per your request. I think :)

marksmithhfx commented:
I am going to leave previous changes as is because I think they are correct. There is no ambiguity so <sessionSavePath> property should be correct.

livecodeali commented:
Sorry, @marksmithhfx, was away for a couple of days! Yes, this looks good, thank you.
@livecode-vulcan review ok c8decb8

livecode-vulcan commented:
💙 review by @livecodeali ok c8decb876c124061e713807872020b9c07c7d632

livecode-vulcan commented:
😎 test success c8decb876c124061e713807872020b9c07c7d632
- try-community-linux-x86: [success](https://vulcan.livecode.com/builders/try-community-linux-x86/builds/2428)
- try-community-mac: [success](https://vulcan.livecode.com/builders/try-community-mac/builds/2438)
- try-community-linux-x86_64: [success](https://vulcan.livecode.com/builders/try-community-linux-x86_64/builds/2421)
- try-community-emscripten: [success](https://vulcan.livecode.com/builders/try-community-emscripten/builds/2439)
- try-community-ios: [success](https://vulcan.livecode.com/builders/try-community-ios/builds/2445)
- try-community-win-x86: [success](https://vulcan.livecode.com/builders/try-community-win-x86/builds/2449)
- try-community-android-armv6: [success](https://vulcan.livecode.com/builders/try-community-android-armv6/builds/2461)
